manifests/filter/cors.pp

```diff
@@ -0,0 +1,94 @@
+# == Resource: repose::filter::ip_identity
+#
+# This is a resource for generating ip identity configuration files
+#
+# === Parameters
+#
+# [*ensure*]
+# Bool.  Ensure config file present/absent
+# Defaults to <tt>present</tt>
+#
+# [*filename*]
+# String.  Config filename
+# Defaults to <tt>cors.cfg.xml</tt>
+#
+# [*allowed_origins*]
+# Required - Hash of Hashes. Allows you to specify which origins are allowed
+# to request resources from this server. Nested hashes have values is_regex
+# valid values: (true | false), and origin, origin can be a regex if is_regex
+# is true, and a full string match of a URL containing port (i.e. 
+# https://www.somehost.com:443) 
+# Defaults to <tt>{ 10 => { 'is_regex' => 'true', 'origin' => '.*' }</tt>
+#
+# [*allowed_methods*]
+# Array of HTTP VERBS. Allows you to specify which HTTP methods are allowed
+# for all resources (when specified under <cross-origin-resource-sharing>).
+# Defaults to <tt>undef</tt>
+#
+# [*resources*]
+# Hash of resources, name of hash being the resource path, hash cintains allowed_methods (Array of HTTP verbs) and comment (XML/HTTP style comment). Specifies
```

Comment:
@bjoshua The documentation specifies that the "resources" parameter should be a hash but the template expects something like this:
```
[
  {
    'name': '/v2.0/tokens',
    'allowed_methods': [
      'POST',
      'GET',
      'OPTIONS',
    ]
  }
]
```

I expected something more like this:

```
{
  '/v2.0/tokens': [
    'POST',
    'GET',
    'OPTIONS',
  ]
}
```